src/containerapp/HISTORY.rst

```diff
@@ -8,6 +8,7 @@ upcoming
 * 'az containerapp session code-interpreter': Fix `--path` in examples
 * 'az containerapp sessionpool create/update': Support `--lifecycle-type` and `--max-alive-period`
 * 'az containerapp env premium-ingress': Deprecate `--min-replicas` and `--max-replicas` parameters, use workload profile scale instead.
+* 'az containerapp session custom-container': Support stop-session for custom container sessions
```

Comment:
whether modeling this as `az containerapp session stop-session` (apply to both Interpreter and CustomContainer) could be a feasible option? I'm thinking that later we might also have stop/delete session for code interpreter (https://learn.microsoft.com/en-us/rest/api/data-plane/containerapps/session-management/delete?view=rest-data-plane-containerapps-2025-02-02-preview&tabs=HTTP), and also list session related commands.

Or sth like `az containerapp session management stop-session`? How would you suggest

Reply:
I think `az containerapp session stop` will be better, have updated the logic